manifests/base/kustomization.yaml

```diff
@@ -9,7 +9,6 @@ resources:
   - jupyter-pytorch-notebook-imagestream.yaml
   - jupyter-tensorflow-notebook-imagestream.yaml
   - jupyter-trustyai-notebook-imagestream.yaml
-  - jupyter-habana-notebook-imagestream.yaml
```

Comment:
Probably this removed accidentaly

Reply:
no it wasnt, it is present on line 7 already.